assets/images/guardians/check_circle.svg

```diff
@@ -0,0 +1,9 @@
+<svg width="78" height="78" viewBox="0 0 78 78" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@gguij004  this is shit, fix this UI when i finish pls

Reply:
Converte it to custom painter :D 🗡️